kolibri/deployment/default/wsgi.py

```diff
@@ -13,6 +13,8 @@
 from django.core.wsgi import get_wsgi_application
 from django.db.utils import OperationalError
 
+import kolibri
```

Comment:
Following our current policy this import should be at the beginning of the file.

Reply:
Good point - possible I didn't have pre-commit installed when I made this commit.